config/configs_basic.yaml

```diff
@@ -71,12 +71,11 @@ timeseries_analyzer:
   max_days: 3600
 
 anovos_basic_report:
-  basic_report: True
+  basic_report: False
   report_args:
     id_col: ifa
     label_col: income
     event_label: '>50K'
-    skip_corr_matrix: True
```

Comment:
@dattranm Once the version of PyPI support this flag, the report config can be added back on this branch as well

Reply:
As above, I think it's better to have a separated PR for config_basics change

Reply:
There is no need to change this, the issue has been fixed months ago. We just don't have a way to install a version that include this fix from PyPI :-)

Reply:
If you want to, you can build the Docker image for the latest main and verify that the current basic_configs.yaml runs fine inside the anovos-worker container.